Makefile

```diff
@@ -1,24 +1,54 @@
-PROJECT_NAME := Pulumi Amazon Web Services (AWS) EKS Components
-SUB_PROJECTS := nodejs/eks
-include build/common.mk
-
-.PHONY: publish_packages
-publish_packages:
-	$(call STEP_MESSAGE)
-	./scripts/publish_packages.sh
-	$$(go env GOPATH)/src/github.com/pulumi/scripts/ci/build-package-docs.sh eks
-
-.PHONY: clean
-clean:
-	rm -rf node_modules package-lock.json yarn.lock
-
-.PHONY: check_clean_worktree
-check_clean_worktree:
-	$$(go env GOPATH)/src/github.com/pulumi/scripts/ci/check-worktree-is-clean.sh
-
-# The travis_* targets are entrypoints for CI.
-.PHONY: travis_cron travis_push travis_pull_request travis_api
-travis_cron: default
-travis_push: only_build check_clean_worktree only_test publish_packages
-travis_pull_request: all
-travis_api: all
+PROJECT_NAME := Pulumi Infrastructure Components for AWS
+
+VERSION         := $(shell pulumictl get version)
+TESTPARALLELISM := 12
```

Comment:
We should keep this is an envvar, as we've had to adjust this dial at times with Travis and would be great to carry over to GHA in case we need it too

Reply:
So this is difficult - GHA has no idea of EnvVars - it has secrets... We can of course make this a secret and then allow the fine tuning??

Reply:
That may be worth considering. We have not had to tweak this recently, but it has been a knob we've used on a handful of occasions with Travis

Reply:
i will keep an eye on this and see what we need to do here...